karinacms/requirements.txt

```diff
@@ -1,6 +1,6 @@
 Django==1.6
```

Comment:
I will give it a try, leave it for now, please continue with the feature that we have discussed.